simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -66,20 +66,20 @@ public PushGateway(String address) {
   }
 
   /**
-   * Pushes all metrics in a registry, replacing all those with the same job as the grouping key.
+   * Pushes all metrics in a registry, replacing all those with the same job and no grouping key.
```

Comment:
There are three other methods down below with similar signature that say "same job and no grouping key". I think this is to differentiate with the methods which take groupingKey map as argument.